auditlog/admin-service/src/main/java/io/debezium/demos/auditing/admin/wih/SendTransactionEventWIHandler.java

```diff
@@ -14,43 +16,41 @@
 import io.debezium.demos.auditing.admin.TransactionData;
 import io.debezium.demos.auditing.admin.TransactionEvent;
 import io.debezium.demos.auditing.admin.VegetableEvent;
-import io.smallrye.reactive.messaging.annotations.Channel;
-import io.smallrye.reactive.messaging.annotations.Emitter;
-import io.smallrye.reactive.messaging.kafka.KafkaMessage;
+import io.smallrye.reactive.messaging.kafka.KafkaRecord;
 
 @ApplicationScoped
 public class SendTransactionEventWIHandler implements WorkItemHandler {
 
     private ObjectMapper json = new ObjectMapper();
-    
+
     @Channel("missingtransactions")
-    Emitter<KafkaMessage<String, String>> emitter;
-    
+    Emitter<KafkaRecord<String, String>> emitter;
```

Comment:
So you cannot do this anymore :-(

Use:

```
Emitter<String> emitter;
```

It will let you send the KafkaRecord or a payload directly.

Reply:
Ok, yeah, but how can we specific key and value?

Reply:
Yes, you can continue sending your KafkaRecord as you did.

Reply:
I.e. how do I send `KafkaRecord` via `Emitter <String>`?

Reply:
Like this:

```
 emitter.send(KafkaRecord.of(key, value));
```

The emitter parameter is the payload type. So you can send the payload directly, or a `Message<X>` like a `KafkaRecord<String, X>`

Reply:
Ok, gotcha. Yes, that works. FWIW, I found the earlier API more comprehensible, for Kafka with its notion of key and value it's quite confusing to have an emitter interface with only one type parameter.

In any case, thanks a lot for the quick reply. Will update this PR now.